src/query/service/src/pipelines/processors/transforms/hash_join/desc.rs

```diff
@@ -30,6 +30,8 @@ use crate::sql::evaluator::Evaluator;
 use crate::sql::executor::HashJoin;
 use crate::sql::plans::JoinType;
 
+pub const MAX_BLOCK_SIZE: usize = 65535;
```

Comment:
why not set the `get_max_block_size` default to 65535?

Reply:
https://github.com/datafuselabs/databend/blob/ac98a1b608c4b23f30ca7e14a90fad652395d059/src/query/settings/src/lib.rs#L124-L133

I think you should test the number table with 65535~

Reply:
`get_max_block_size` is also used by `insert`. If we change it to 65535 as default, may influent query performance.

For join scenarios, we've proven that `65535` has performance gains.

Reply:
we should rename it to `JOIN_BATCH_SIZE` to avoid confusion.